streams.go

```diff
@@ -82,3 +87,20 @@ func (c *Client) GetFollowedStream(params *FollowedStreamsParams) (*StreamsRespo
 
 	return streams, nil
 }
+
+// GetStreamKey : Returns the secret stream key of the broadcaster
+//
+// Required scope: channel:read:stream_key
+func (c *Client) GetStreamKey(params *StreamKeyParams) (*StreamsResponse, error) {
+	resp, err := c.get("/streams/key", &ManyStreams{}, params)
+	if err != nil {
+		return nil, err
+	}
+
+	streams := &StreamsResponse{}
+	resp.HydrateResponseCommon(&streams.ResponseCommon)
+	streams.Data.Streams = resp.Data.(*ManyStreams).Streams
```

Comment:
`ManyStreams` is not a correct representation of the response object for this endpoint. The response, according to the [docs](https://dev.twitch.tv/docs/api/reference#get-stream-key), should only in the `stream_key` property.

```json
{
  "data": [
    {
      "stream_key": "live_44322889_a34ub37c8ajv98a0"
    },
  ]
}
```

My suggestion would be to create a new struct called `ManyStreamKeys` instead. For example:

```go
type ManyStreamKeys struct {
    Data []struct{
        StreamKey string `json:"stream_key"`
    } `json:"data"`
}
```